lib/lambda_ethereum_consensus/execution/engine_api/mocked.ex

```diff
@@ -20,6 +20,6 @@ defmodule LambdaEthereumConsensus.Execution.EngineApi.Mocked do
 
   @spec forkchoice_updated(map, map | any) :: {:ok, any} | {:error, any}
   def forkchoice_updated(_forkchoice_state, _payload_attributes) do
-    {:ok, %{"payload_id" => nil, payload_status: %{"status" => "SYNCING"}}}
+    {:ok, %{"payload_id" => nil, "payload_status" => %{"status" => "SYNCING"}}}
```

Comment:
Shouldn't we return this already parsed?

Reply:
Yes, that's what I mean. It seems I mistook the `EngineApi` and `ExecutionClient` modules 😭